### PR TITLE
fix: single storages loader

### PR DESCRIPTION
**Pull request by olenagerasimova**
part of #1237 

Create storages loader instance only once in artipie/artipie

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus please, review, just a view changes

**Review by dgarus — DISMISSED** [review]
thanks!

File: src/main/java/com/artipie/http/DockerRoutingSlice.java [hunk 1]
```diff
@@ -63,10 +71,20 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
...
             if (group.isEmpty() || group.equals("/")) {
-                rsp = new BasicAuthSlice(
+                rsp = new BasicAuthzSlice(
                     new BaseEntity(),
                     this.settings.authz().authentication(),
-                    user -> !user.equals(Authentication.ANY_USER)
+                    new OperationControl(
+                        user -> {
+                            // @checkstyle NestedIfDepthCheck (10 lines)
+                            PermissionCollection res = new FreePermissions();
```
Comment (dgarus) [inline src/main/java/com/artipie/http/DockerRoutingSlice.java:80]: Can't understand what does this mean?

Comment (olenagerasimova) [reply src/main/java/com/artipie/http/DockerRoutingSlice.java:80]: the same as was `user -> !user.equals(Authentication.ANY_USER)`:
`v2` endpoint (docker root) is available for any authenticated user

Comment (dgarus) [reply src/main/java/com/artipie/http/DockerRoutingSlice.java:80]: > the same as was `user -> !user.equals(Authentication.ANY_USER)`: `v2` endpoint (docker root) is available for any authenticated user

This means if the user is authenticated, he has the permission to pull any image. Right?

**Pull request merged**